tests/crustal_model_3D.prm

```diff
@@ -0,0 +1,131 @@
+
+############### Global parameters
+
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set CFL number                             = .5
+set Linear solver tolerance                = 1e-9
+set Nonlinear solver scheme                = iterated Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 3
```

Comment:
The only issue I can envision is if a later change allows the model to converge in less than 3 iterations. Is there a reason to do more than one non-linear iteration? If so, perhaps set the Nonlinear solver tolerance higher so that it would never reach 1e-5? However, this is very likely a non-issue and an easy fix later on.